.ci/azure/linux.yml

```diff
@@ -105,6 +110,29 @@ jobs:
     workingDirectory: $(BUILD_DIR)
     displayName: 'Build Lin'
 
+  - script: >
```

Comment:
@azhogov Do you mean this task `- task: CMake@1` ?
But in this case it will require both java available on azure machine and cuda ...

In our cmake build we disable java_api

Reply:
I see. In general, all plugins should be built by one command.
You split the jobs as intermediate step, ok.